src/nativestore/NodeManager.cpp

```diff
@@ -25,13 +25,27 @@ limitations under the License.
 #include "PropertyLink.h"
 #include "RelationBlock.h"
 #include "iostream"
+#include <sys/stat.h>
 
 Logger node_manager_logger;
 pthread_mutex_t lockEdgeAdd;
 
+bool fileExists(const std::string& path) {
+    std::ifstream file(path);
+    return file.is_open();
+}
+
+std::fstream* openFile(const std::string &path, std::ios_base::openmode mode) {
```

Comment:
Its better to move this function to utils. Might be useful in other files as well

Reply:
Done.

Reply:
Ok